amundsen_common/elastic_search/elastic_search_index.py

```diff
@@ -0,0 +1,90 @@
+import textwrap
+
+# Specifying default mapping for elasticsearch index
+# Documentation: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html
+# Setting type to "text" for all fields that would be used in search
+# Using Simple Analyzer to convert all text into search terms
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-simple-analyzer.html
+# Standard Analyzer is used for all text fields that don't explicitly specify an analyzer
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-analyzer.html
+DEFAULT_ELASTICSEARCH_INDEX_MAPPING = textwrap.dedent(
```

Comment:
should we call TABLE_INDEX_MAP also should we copy https://github.com/lyft/amundsensearchlibrary/blob/master/search_service/models/index_map.py#L71 as well?

I think it needs to bump the setup.py version as well.

@jinhyukchang do you think we should move it under model instead of having a different module path?

Reply:
+1 done to both, tyvm for your help!

Reply:
> should we call TABLE_INDEX_MAP also should we copy https://github.com/lyft/amundsensearchlibrary/blob/master/search_service/models/index_map.py#L71 as well?
> 
> I think it needs to bump the setup.py version as well.
> 
> @jinhyukchang do you think we should move it under model instead of having a different module path?

@feng-tao Good point about having it under model.
I think I am fine with current approach. May be we can have factory function (Dashboard is coming shortly) so that it can just return `MAPPING` based on the type?